docs/index.rst

```diff
@@ -3,22 +3,66 @@
 
       SPDX-License-Identifier: Apache-2.0
 
-Welcome to Unidist's documentation!
-===================================
+What is unidist?
+""""""""""""""""
 
-The framework is intended to provide the unified API for distributed execution
-by supporting various distributed task schedulers. At the moment the following schedulers
-are supported under the hood:
+unidist (`Unified Distributed Execution`) provides the unified API for distributed execution by supporting various performant execution backends.
+At the moment the following backends are supported under the hood:
 
 * `Ray`_
+* `MPI`_
 * `Dask Distributed`_
 * `Python Multiprocessing`_
-* `MPI`_
 
 Also, the framework provides a sequential :doc:`Python backend <flow/unidist/core/backends/python/backend>`,
-that can be used for debug purposes.
+that can be used for debugging.
+
+unidist is designed to work in a `task-based parallel model`_. The framework mimics `Ray`_ API and expands the existing frameworks
+(Ray and Dask Distributed) with additional features.
+
+Installation
+============
+
+unidist can be installed from sources using ``pip``:
+
+.. code-block:: bash
+
+   # Dependencies for all the execution backends will be installed
+   $ pip install git+https://github.com/modin-project/unidist#egg=unidist[all]
+
+For more information about installation and supported OS platforms see the installation page.
+
```

Comment:
For now yes, but this is draft for future page which is also important as others docs. It would be nice to include this page right now to highlight  installation tips and OS support for backends.

Reply:
I suggest adding this sentence when we add installation page.

Reply:
Ok, removed